src/compinterpret.py

```diff
@@ -527,14 +528,12 @@ def Varable_Declaration_Stmt(self):
         self.file.pop()
         self.js += f'assign(\"{var_name}\", '
         if self.Expr():  # inserts expression
-            success, priority = self.EndStmt()
+            success, priority = self.EndStmt(f'{keyword == "let"}, {{}}, {lifetime})')
```

Comment:
here we pass a template string for the end statement compiler to insert the priority value
it's either we pass a template string in, or move the js addition part out of `EndStmt()`